flink-cmd/pom.xml

```diff
@@ -23,14 +23,14 @@
   <parent>
     <artifactId>zeppelin-interpreter-parent</artifactId>
     <groupId>org.apache.zeppelin</groupId>
-    <version>0.10.0-SNAPSHOT</version>
+    <version>0.9.1-SNAPSHOT</version>
```

Comment:
This change is not necessary. We use 0.9.1-SNAPSHOT for branch-0.9, and 0.10.0-SNAPSHOT for master

Reply:
Ah, should parents refer to version 0.10.0 in both flink-cmd and spark-submit? At least I am not able to build that way.

Reply:
No, all the versions should be 0.9.1-SNAPSHOT, do you mean you unable to build branch-0.9 ?

Reply:
Yes, I am building on branch-0.9. Should I target this bug fix to master instead?

Reply:
Yes, usually we create PR based on master branch and will cherry-pick that into branch-0.9 when merging.

Reply:
Ok, I changed to master.